cristin-person/src/test/java/no/unit/nva/cristin/person/update/cristinPersonUpdate.feature

```diff
@@ -39,12 +40,12 @@ Feature: API tests for Cristin Person Update
     Then status 401
     And match response.message == 'Unauthorized'
 
-  Scenario: Update returns status 403 Forbidden when not authorized
+  Scenario: Update returns status 401 Unauthorized when not authorized because of getPersonCristinId
```

Comment:
Add a karate test where you mock getPersonCristinId using HandlerRequestBuilder and see if you get a forbidden after that when you dont have required access rights. Maybe in separate PR if you want to get this done.